server/src/services/session.service.ts

```diff
@@ -19,6 +21,25 @@ export class SessionService {
     this.access = AccessCore.create(accessRepository);
   }
 
+  async handleCleanup() {
+    const sessions = await this.sessionRepository.search({
+      updatedBefore: DateTime.now().minus({ days: 90 }).toJSDate(),
```

Comment:
Do we want to make the timeout configurable?

Reply:
Maybe?

Reply:
@bo0tzz where would you put that in the admin settings? User Settings?

Reply:
We maybe should move password and auth and oauth into a parent auth category in the UI.

Reply:
An overarching auth section is probably the cleanest, but under server or user settings would work for me as well.

Reply:
Moving the auth/oauth settings around and adding the delay for deleting sessions will be its own PR